### PR TITLE
Add working webpack-configured frontend with typescript

### PR DESCRIPTION
**Pull request by hamidaebadi**
Please view this pull-request and merge it if everything seems to be OK.
**What happens after merging this pull-request?**
After merging this pull-request you can run the frontend by typing `npm run start:frontend` within the project's root directory.
For building the frontend run command`npm run build` in the root directory of the project.

## Sub-task of

#4 Frontend skeleton

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -80,7 +80,6 @@ typings/
 
 # Nuxt.js build / generate output
 .nuxt
-dist
```
Comment (nualn) [inline .gitignore:83]: Buildi pois versionhallinnasta

File: package.json [hunk 2]
```diff
@@ -5,13 +5,16 @@
   "private": true,
   "scripts": {
     "start": "npm run start:backend & npm run start:frontend",
-    "build": "npm -ws --if-present run build",
+    "build": "npm -w @lentovaraukset/frontend run build",
```
Comment (nualn) [inline package.json:8]: Aikaisempi komento ajoi buildin myös backendille. Jos haluaa ajaa buildin vain frontille, voisi olla parempi lisätä build:frontend skripti.

File: .gitignore [hunk 3]
```diff
@@ -102,5 +102,3 @@ dist
 
 # TernJS port file
 .tern-port
-
-build
```
Comment (nualn) [inline .gitignore:106]: build-kansio kannattaa pitää gitignoressa, koska backendin buildi menee tällä hetkellä backend/build kansioon.

Comment (hamidaebadi) [reply .gitignore:106]: done

File: package.json (hunk 2 above)
Comment (hamidaebadi) [reply package.json:8]: done

**Review by Jokauppi — APPROVED** [review]

**Pull request merged**